android-test/build.gradle.kts

```diff
@@ -20,7 +20,8 @@ android {
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     testInstrumentationRunnerArguments += mapOf(
       "runnerBuilder" to "de.mannodermaus.junit5.AndroidJUnit5Builder",
-      "notPackage" to "org.bouncycastle"
+      "notPackage" to "org.bouncycastle",
+      "configurationParameters" to "junit.jupiter.extensions.autodetection.enabled=true"
```

Comment:
Do our end-users need to do this? JUnit 5 is such a mess.

Reply:
don't need to, users have the option of
 * using the extension directly by annotating test class.
 * using the extension directly by annotating an annotation to collect extensions
 * setting this property to magically apply extension to all tests

Reply:
Oh, just realized that this is in internal package, but it's public API, as users can choose how to use the extensions (you choose to use the atuodetection in this projecet). Could this be made actual API? @swankjesse

Reply:
I think we should change our MockWebServer JUnit 5 extension API to use an annotation on each annotated class. It’s two imports instead of JUnit 4’s single import, but it’s self-contained in the test class.